packages/ui-rating/src/Rating/theme.js

```diff
@@ -22,8 +22,17 @@
  * SOFTWARE.
  */
 
-export default function ({ breakpoints }) {
+/**
+ * Generates the theme object for the component from the theme and provided additional information
+ * @param  {Object} theme The actual theme object.
+ * @param  {Object} themeOverride User provided overrides of the default theme mapping.
+ * @return {Object} The final theme object with the overrides and component variables
+ */
+const generateComponentTheme = (theme, themeOverride = {}) => {
   return {
-    contentMaxWidth: breakpoints.large
+    contentMaxWidth: theme['breakpoints']['large'],
+    ...themeOverride
   }
 }
+
+export default generateComponentTheme
```

Comment:
This is some weird stuff, as I can tell the original code was doing nothing

Reply:
Hm indeed, the code doesn't seem to use it anywhere. I think we should delete it then.

Reply:
yea, I double checked, it does nothing, ill delete it